jdaviz/configs/imviz/plugins/coords_info/coords_info.vue

```diff
@@ -1,6 +1,23 @@
 <template>
-    <div v-if="pixel" style="white-space: nowrap;">
-        <b v-if="pixel">Pixel </b>{{ pixel }}&nbsp;&nbsp;<b v-if="value">Value </b>{{ value }}<br>
-        <b v-if="world">World </b>{{ world }}<br>
-    </div>
+  <div v-if="pixel" style="white-space: nowrap;">
+    <table>
```

Comment:
My vuejs-fu is too weak. I think this accomplishes what you asked for, but I don't know how robust it is across browser and monitor settings...

Reply:
Looks good to me.  At small widths, it overlaps the plugin menu button, but that button isn't clickable while mousing over the image anyways.  We have another ticket to eventually deal with minimum widths and overflowing across the app, so I don't think you need to worry about that too much right now.